hydrate/background.js

```diff
@@ -1,19 +1,36 @@
 const toggle = {
     status: true
 }
-const switchToggle = function setToggleFalse(toggle){
-    if (toggle.status === true){
+
+const switchToggle = function setToggleFalse(toggle) {
+    if (toggle.status === true) {
+        chrome.browserAction.setIcon({
+            path : {
+                16: "icons/icon-gray16.png",
+                32: "icons/icon-gray32.png",
+                48: "icons/icon-gray48.png",
+                128: "icons/icon-gray128.png"
+            }
+        });
         toggle.status = false
-        console.log("Stopped extension")
-        alert("Stopped Hydrate!\nClick again to restart")
```

Comment:
Why you put a semicolon at the end of some lines?

Reply:
Force of habit, used to adding semicolon to javascript

Reply:
I'll remove them.

See any other issues?